### ISSUE TITLE
Inconsistent output between PyTorch and LibTorch

### ISSUE THREAD

**Issue by zhihangxie** [pytorch/pytorch#62252]
## 🐛 Bug

Hi, I created a Conv1d TorchScript module in PyTorch via torch.jit.script, but got inconsistent output in LibTorch. Could you kindly check if there is anything wrong? Thanks.

## To Reproduce

1. download and unzip the TorchScript module [here](https://github.com/pytorch/pytorch/files/6883692/script_conv.zip)
2. download and unzip the input data [here](https://github.com/pytorch/pytorch/files/6883712/input.zip)
3. PyTorch (1.9.0)
>>> import torch
>>> import numpy as np
>>> module = torch.jit.load('script_conv.pt')
>>> input = np.fromfile('input.npy', dtype=np.float32)
>>> input = input.reshape((1, 80, 269))
>>> output = module(input)
>>> output[0, 0, :10]  => [1.9974, 1.2362, 1.5868, 1.3649, 1.1560, 1.2611, 1.4425, 1.6751, 1.7646, 1.4132]
4. LibTorch (1.9.0)
>>> int main()
>>> {
>>>     torch::jit::script::Module module = torch::jit::load("script_conv.pt");
>>> 
>>>     std::fstream ifs("input.npy");
>>>     float matrix[1][80][269] = {0};
>>>     ifs.read(reinterpret_cast<char*>(&matrix), sizeof(matrix));
>>>     torch::Tensor input = torch::from_blob(fmatrix, { 1, 80, 269 });
>>> 
>>>     at::Tensor output = module.forward({ input }).toTensor();
>>>     // output[0, 0, :10] => [0.814523, 0.978139, 0.987679, 1.08601, 1.08149, 1.11269, 1.12558, 1.16961, 1.18193, 1.17423]
>>> }

## Expected behavior

LibTorch's output is expected to be the same as PyTorch's output.

## Environment

 - PyTorch Version (e.g., 1.0): 1.9.0
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.8.5
 - CUDA/cuDNN version: 11.1
 - GPU models and configuration: Conv1d(80, 512, 7, 1, padding=3)
 - Any other relevant information: None

## Additional context

None

cc @gmagogsfm

**Comment by zhihangxie** [comment]
It turns out to be a platform-dependent issue, so close the issue. Thanks @mruberry and @gmagogsfm.
